tests/dummy/app/templates/demo/progress.hbs

```diff
@@ -1,52 +1,105 @@
-    {{form.element controlType="number" label="Max" property="maxValue"}}
-    {{#form.element controlType="power-select" options=typeChoices label="Type" property="type" as |el|}}
-      {{el.control searchEnabled=false}}
-    {{/form.element}}
-    {{form.element controlType="checkbox" label="Show label" property="showLabel"}}
-    {{form.element controlType="number" label="Round to digits" property="roundDigits"}}
-    {{form.element controlType="checkbox" label="Striped" property="striped"}}
-    {{form.element controlType="checkbox" disabled=notStriped label="Animate" property="animate"}}
-  {{/bs-form}}
+    <form.element @controlType="checkbox" @label="Striped" @property="striped" />
+    <form.element
+      @controlType="checkbox"
+      @disabled={{notStriped}}
+      @label="Animate"
+      @property="animate"
+     />
+
+  </BsForm>
   <hr>
   <!-- BEGIN-SNIPPET progress-options -->
-  {{#bs-progress as |p|}}
-    {{p.bar value=value minValue=minValue maxValue=maxValue showLabel=showLabel roundDigits=roundDigits type=type striped=striped animate=animate}}
-  {{/bs-progress}}
+  <BsProgress as |pg|>
+    <pg.bar
+      @value={{value}}
+      @minValue={{minValue}}
+      @maxValue={{maxValue}}
+      @showLabel={{showLabel}}
+      @roundDigits={{roundDigits}}
+      @type={{type}}
+      @striped={{striped}}
+      @animate={{animate}}
+     />
+
+  </BsProgress>
   <!-- END-SNIPPET -->
 </div>
 <div class="highlight">
-  {{code-snippet name="progress-options.hbs"}}
+  <CodeSnippet @name="progress-options.hbs" />
 </div>
-
-<h2>Stacked</h2>
-
+<h2>
+  Stacked
+</h2>
 <div class="bs-example">
   <!-- BEGIN-SNIPPET progress-stacked -->
-  {{#bs-progress as |p|}}
-    {{p.bar value=value minValue=minValue maxValue=maxValue showLabel=showLabel roundDigits=roundDigits type=type striped=striped animate=animate}}
-    {{p.bar value=20 minValue=minValue maxValue=maxValue showLabel=showLabel roundDigits=roundDigits type="warning" striped=striped animate=animate}}
-    {{p.bar value=5 minValue=minValue maxValue=maxValue showLabel=showLabel roundDigits=roundDigits type="danger" striped=striped animate=animate}}
-  {{/bs-progress}}
+  <BsProgress as |pg|>
+    <pg.bar
+      @value={{value}}
+      @minValue={{minValue}}
+      @maxValue={{maxValue}}
+      @showLabel={{showLabel}}
+      @roundDigits={{roundDigits}}
+      @type={{type}}
+      @striped={{striped}}
+      @animate={{animate}}/>
+    <pg.bar
+      @value="20"
+      @minValue={{minValue}}
+      @maxValue={{maxValue}}
+      @showLabel={{showLabel}}
+      @roundDigits={{roundDigits}}
+      @type="warning"
+      @striped={{striped}}
+      @animate={{animate}}/>
+    <pg.bar
+      @value="5"
+      @minValue={{minValue}}
+      @maxValue={{maxValue}}
+      @showLabel={{showLabel}}
+      @roundDigits={{roundDigits}}
+      @type="danger"
+      @striped={{striped}}
+      @animate={{animate}}/>
+  </BsProgress>
```

Comment:
This is still failing with a template compiler error for Ember 2.18, no idea really why?
https://travis-ci.org/kaliber5/ember-bootstrap/jobs/528738875#L662

Reply:
Ah, I don't know why, but I was able to fix this by putting the closing `/>` on the following line. Same for the other two `pg.bar`s.

Pushed to your branch, should hopefully get green now...